dubbo-demo/dubbo-demo-spring-boot/dubbo-demo-spring-boot-consumer/pom.xml

```diff
@@ -140,9 +140,18 @@
         </dependency>
         <dependency>
             <groupId>io.micrometer</groupId>
-            <artifactId>micrometer-tracing-bridge-otel</artifactId>
+            <artifactId>micrometer-tracing-bridge-brave</artifactId>
```

Comment:
Out of curiosity - why the change?

Reply:
I tested on the demo, but I forgot to rollback

Reply:
No problem, it doesn't matter which one you're using